src/main/kotlin/uk/gov/dluhc/printapi/dto/IssuerDto.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.dluhc.printapi.dto
+
+data class IssuerDto(
```

Comment:
Possibly this could be `LocalAuthorityDto` but not 100% sure that it will always only contain details related to the LA and never an ERO.